### PR TITLE
docs(message): add example of rich text content to stories FE-3139

### PR DESCRIPTION
**Pull request by edleeks87**
### Proposed behaviour

Add example of `Message` rendering rich content to stories

### Current behaviour

No example of how to render rich content within the `Message` component
### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### Additional context

image

### Testing instructions

http://localhost:9001/?path=/docs/message--with-list-content

or 

https://5ecf782fe724630022d27d7d-lmtyxklhui.chromatic.com/?path=/docs/message--default-story

### CODE REVIEW

File: src/components/message/message.stories.mdx [hunk 1]
```diff
@@ -120,6 +123,36 @@ Message variant is `info` by default
   </Story>
 </Preview>
 
+### With rich content
+It is also possible to use the `List`, `ListItem` and `Typography` components to render content other than plain text in a `Message`.
+<Preview>
+  <Story name="with list content">
```
Comment (DipperTheDan) [inline src/components/message/message.stories.mdx:129]: Title is `rich content` but story name is `list content`. Shouldn't these be the same?

Comment (edleeks87) [reply src/components/message/message.stories.mdx]: Yeah good spot, I decided to make the title more general but forgot to update the name

**Review by DipperTheDan — APPROVED** [review]

**Review by dawidzarzycki — APPROVED** [review]
All about the code looks fine to me, I would think about changing `local path` in `testing instructions` to maybe chromatic link? https://5ecf782fe724630022d27d7d-lmtyxklhui.chromatic.com/?path=/docs/message--default-story
It's up to you

**Review by KatarzynaQA — APPROVED** [review]

**Pull request merged**